tests/conftest.py

```diff
@@ -29,6 +37,17 @@ def client():
     return TestClient(app)
 
 
+@pytest.fixture(scope='module')
+def client_with_logger():
```

Comment:
Maybe add a seperate file for all log fixtures?

Reply:
There are only two log fixtures, I dont think that's enough to add a separate file.

Reply:
Do you think that most of the tests need the log fixtures? If so, maybe it's a good choice to keep it here.
But if not, please move it to a less general file :)